src/server/ocpi/ocpi-services-impl/ocpi-2.1.1/OCPIMapping.ts

```diff
@@ -164,10 +165,29 @@ export default class OCPIMapping {
     const sites = await SiteStorage.getSites(tenant.id, { issuer: true, withOnlyChargingStations: true }, { limit, skip });
     // Convert Sites to Locations
     for (const site of sites.result) {
-      ocpiLocationsResult.result.push(await OCPIMapping.convertSite2Location(tenant, site, options));
+      let foundPublicChargingStation = false;
+      // Site with Charging Station only
+      const siteAreas = await SiteAreaStorage.getSiteAreas(tenant.id, { withChargingStations: true }, Constants.DB_PARAMS_MAX_LIMIT);
```

Comment:
You can return only site areas with stations inside: `{ withChargingStations: true,  withOnlyChargingStations: true }`